docs/admin/integrations/vsphere_setup.md

```diff
@@ -0,0 +1,122 @@
+| Sync Tagged Only              | True                                                                           | Only take into consideration tagged VMs (on the Nautobot side) when performing the sync.                      |
+| Virtual Machine Status Map    | `{"POWERED_OFF": "Offline", "POWERED_ON": "Active", "SUSPENDED": "Suspended"}` | Maps vSphere Virtual Machine status to Nautobot status.                                                       |
+| Virtual Machine IP Status Map | `{"PREFERRED": "Active", "UNKNOWN": "Reserved"}`                               | Maps vSPhere IP status to Nautobot status                                                                     |
+| Virtual Machine Interface Map | `{"CONNECTED": true, "NOT_CONNECTED": false}`                                  | Maps vSphere interface state to boolean values.                                                               |
+| Default Cluster Group Name    | vSphere Default Cluster Group                                                  | Denotes the default Cluster Group name used in the sync if `Use Clusters` is set to False.                    |
+| Default Cluster Name          | vSphere Default Cluster                                                        | Denotes the default Cluster name used in the sync if `Use Clusters` is set to Fals.e.                         |
+| Default Cluster Type          | VMWare vSphere                                                                 | Denotes the default Cluster Type to set in the sync.                                                          |
+
+Each vShere configuration must be linked to an [External integration](https://docs.nautobot.com/projects/core/en/stable/user-guide/platform-functionality/externalintegration/?h=external+int) describing the vSphere instance. The following External Integration fields must be defined for integration to work correctly:
+
+| Setting       | Description                                                                      |
+| :------------ | :------------------------------------------------------------------------------- |
+| Remote URL    | URL of the remote vSphere instance to sync with.                                 |
+| Verify SSL    | Toggle SSL verification when sycning data with vSphere                           |
+| Secrets Group | Secrets Group defining credentials used when connecting to the vSphere instance. |
+| Timeout       | How long HTTP requests to vSphere should wait for a response before failing.     |
+
+The Secrets Group linked to the vSphere External Integration must contain password and username secrets defined as per the below:
+
+| Access Type | Secret Type |
+| :---------- | :---------- |
+| REST        | Password    |
+| REST        | Username    |
+
+
+### Configuring Virtual Machine Status Map
+
+Virtual Machine Status Map is a mandatory setting used to map the status of a machine in vSphere to a status in Nautobot.
```

Comment:
```suggestion
Virtual Machine Status Map is a mandatory setting used to map the status of a machine in vSphere to a Status in Nautobot.
```

Reply:
Status is a proper name of an object in Nautobot so we should always capitalize it.